upgrade/client.py

```diff
@@ -35,27 +44,34 @@ def satellite6_client_setup():
         The Docker VM IP/Hostname on rhevm to create clients
     """
     # If User Defined Clients Hostname provided
-    clients6 = os.environ.get('CLIENT6_HOSTS')
-    clients7 = os.environ.get('CLIENT7_HOSTS')
+    clients6 = CLIENT6_HOSTS
+    clients7 = CLIENT7_HOSTS
     puppet_clients6 = puppet_clients7 = None
-    docker_vm = os.environ.get('DOCKER_VM')
-    clients_count = os.environ.get('CLIENTS_COUNT')
-    from_version = os.environ.get('FROM_VERSION')
+    docker_vm = DOCKER_VM
+    clients_count = CLIENTS_COUNT
+    from_version = FROM_VERSION
     sat_host = env.get('satellite_host')
     if clients6:
         clients6 = [client.strip() for client in str(clients6).split(',')]
         # Sync latest sat tools repo to clients if downstream
-        if os.environ.get('TOOLS_URL_RHEL6'):
+        if settings.REPOS.SATTOOLS_REPO.RHEL6:
```

Comment:
```suggestion
        if settings.repos.sattools_repo.rhel6
```
Use all lower case ! Everywhere !